code/datums/components/chasm.dm

```diff
@@ -23,7 +23,8 @@
 		/obj/effect/collapse,
 		/obj/effect/particle_effect/ion_trails,
 		/obj/effect/dummy/phased_mob,
-		/obj/effect/mapping_helpers
+		/obj/effect/mapping_helpers,
+		/obj/effect/wisp
```

Comment:
```suggestion
		/obj/effect/wisp,
```

Trailing comma on the last one, keeps the diffs lighter.

Reply:
we need this on contribution guidelines if like 3 of us are enforcing it